package.json

```diff
@@ -207,7 +207,8 @@
   },
   "scripts": {
     "----------------------------------------- static analysis -----------------------------------------": "",
-    "lint": "eslint lib test --quiet --fix",
+    "lint": "eslint . --fix --report-unused-disable-directives",
+    "lint-no-fix": "eslint . --quiet --fix --report-unused-disable-directives",
```

Comment:
I remember that lint-no-fix did at some point indeed remove the --fix flag, but for some reason it is here. @ephys can you fix that when you implement my suggestions? (btw, I can also make a PR for my suggestions if you prefer. I can imagine that it's not the thing people most like to work on)

Reply:
That's weird. I must have added it back by accident when changing something else

If you're willing to take the time to do theses changes, please do! Otherwise I'll include it in my PR that makes the linter run on markdown too :)

Reply:
Then I'll make a PR with my suggestions later this week, so you can work on the markdown. You probably already thought of it, but do check how the linted markdown files look with docusaurus and combine those PRs instead of checking it with our current docs generation

Reply:
Iirc @fncolon added the --fix semi-recently

Reply:
No, it's not about that. This PR added the lint-no-fix but it accidentally still has the fix flag so it's doing what is intended. And at some point in this PR there was a commit that did not have the fix flag in lint-no-fix as far as I can remember so was just something I noticed. But will be fixed later this week